grails-app/conf/logback.xml

```diff
@@ -1,40 +1,37 @@
 <configuration>
 
-    <!--
-       - Connect Sentry with our logging framework.
-       -
-       - Most of Sentry's configuration is done programmatically;
-       - cf. SentryInterceptor and SentryServletContainerInitializer.
-      -->
+    <!-- Allows Logback logs to appear in Sentry. See sentry.properties for configuration. -->
     <appender name="sentry" class="io.sentry.logback.SentryAppender">
         <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
             <level>info</level>
         </filter>
-        <!--
-           - When we log an error or exception, send recent log entries
-           - at level 'info' or higher to Sentry for additional context.
-          -->
-        <minimumEventLevel>error</minimumEventLevel>
-        <minimumBreadcrumbLevel>info</minimumBreadcrumbLevel>
+        <encoder>
+            <pattern>
+                <!-- https://logback.qos.ch/manual/layouts.html -->
+                %date{ISO8601} %-5level [%thread] %logger{40}: %message%n%xException
+            </pattern>
+        </encoder>
     </appender>
 
+    <!--  Outputs Logback logs to the console.  -->
     <appender name="stdout" class="ch.qos.logback.core.ConsoleAppender">
+        <!-- Adds colors to the logs only if the terminal can support it https://logback.qos.ch/manual/layouts.html#coloring -->
+        <withJansi>true</withJansi>
         <encoder>
             <pattern>
-                %date{ISO8601} %5level [%-10.10thread] %-40.40logger{40}: %message%n%xException
+                <!-- https://logback.qos.ch/manual/layouts.html -->
+                %cyan(%date{ISO8601}) %highlight(%-5level) %magenta([%thread]) %yellow(%logger{40}): %message%n%xException
```

Comment:
I might have been wrong about the color issue. Here's the PR I was referring a few weeks ago re: "Matt removed something fancy that was causing issues on Bamboo". Turns out it was due to webpack, not our logging configuration. 

https://github.com/openboxes/openboxes/pull/3052